### ISSUE TITLE
Renew Tugboat TLS certificate ~December 2023 or January 2024

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#13632]
## Description

Hoping that we search for this ticket rather than create a new ticket and lose some context.

A new TLS certificate should be created in Venafi to use for the Tugboat server.

This should include `training.cms.va.gov` so that we don't forget about it and have to renew it with less than a day of warning.

Ask me why this comes to mind.

## Acceptance Criteria

- [x] Tugboat TLS certificate is renewed.
- [x] Certificate has a SAN of `training.cms.va.gov`.

**Comment by EWashb** [comment]
This is blocked by the current sprint Venafi work, right @ndouglas?

**Comment by ndouglas** [comment]
@EWashb Not blocked because both Eric and I can do this and only one of us needs to be available and conscious.

**Comment by ndouglas** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Accessibility-Issues-62b4be010177900017e82254/issues/department-of-veterans-affairs/va.gov-cms/13632?planning-poker) with Zenhub @olivereri @ariperez @teeshe @JunTaoLuo @edmund-dunn

**Comment by ndouglas** [comment]
![Screenshot 2023-12-27 at 1 26 32 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/1318579/27eae8f9-99a7-4caf-8fd9-93d4ef90c50f)
